README.md

```diff
@@ -1,76 +1,61 @@
 # nvim-lspconfig
 
-Collection of common configurations for the [Nvim LSP client](https://neovim.io/doc/user/lsp.html).
-
-## Best-effort, unsupported
-
+Collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
 The configurations here are strictly **best effort and unsupported**.
```

Comment:
"unsupported" by whom? Clearly they are supported by the contributors to this repository ;)

Maybe what is meant is "are supported on a best-effort basis", meaning that *server* configuration changes may not be picked up unless someone notices, reports, and suggest fixes for them?

Reply:
Changed, also streamlined the intro. The configurations are reasonable well supported now anyways.